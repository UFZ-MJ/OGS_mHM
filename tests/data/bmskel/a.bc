#BOUNDARY_CONDITION
 $PCS_TYPE
  HEAT_TRANSPORT
 $PRIMARY_VARIABLE
  TEMPERATURE1
 $GEO_TYPE
  POINT POINT0
 $DIS_TYPE
  CONSTANT 274
#STOP
