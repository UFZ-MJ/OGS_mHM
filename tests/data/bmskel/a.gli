#POINTS
 0 0 0 0 $NAME POINT0
 1 1 0 0
#STOP
