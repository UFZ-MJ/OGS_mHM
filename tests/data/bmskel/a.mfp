#FLUID_PROPERTIES                                                          
 $FLUID_TYPE                                                               
  LIQUID                                                                   
 $PCS_TYPE                                                                 
  PRESSURE1                                                                
 $DENSITY                                                                  
  1 1000.0                                                                 
 $VISCOSITY                                                                
  1 0.0                                                                    
 $SPECIFIC_HEAT_CAPACITY                                                   
  1 0.0                                                                    
 $HEAT_CONDUCTIVITY                                                        
  1 0.0                                                                    
#STOP