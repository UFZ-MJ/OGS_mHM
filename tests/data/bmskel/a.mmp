#MEDIUM_PROPERTIES
 $GEOMETRY_DIMENSION
  1
 $GEOMETRY_AREA
  1.0
#STOP
 $POROSITY
  1 0.19
 $TORTUOSITY
  1 1.0
 $HEAT_DISPERSION
  1 0.001 0.001
 $DIFFUSION
  273