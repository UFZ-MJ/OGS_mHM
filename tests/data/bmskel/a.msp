#SOLID_PROPERTIES
 $DENSITY
  1 2000.0
  CAPACITY
  1 900
  CONDUCTIVITY
  1 5
#STOP
