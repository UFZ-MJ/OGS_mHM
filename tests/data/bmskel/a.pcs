#PROCESS
 $PCS_TYPE
  HEAT_TRANSPORT
 $NUM_TYPE
  NEW
#STOP
