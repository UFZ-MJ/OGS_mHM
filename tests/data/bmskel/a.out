#OUTPUT
 $PCS_TYPE
  HEAT_TRANSPORT
 $NOD_VALUES
  TEMPERATURE1
 $GEO_TYPE
  POINT POINT0
 $DAT_TYPE
  TECPLOT
 $TIM_TYPE
  1
#STOP
